web-docs/docs/02-CHANGELOG.mdx

```diff
@@ -23,7 +32,8 @@ toc_max_heading_level: 2
 
 ## Version 1.15.3
 ### Features
-- Added service timer to the application reboot watchdog to ensure application is in a stable state before allowing the watchdog to be serviced.
+- Added service timer to the application reboot watchdog to ensure application is in a stable state
```

Comment:
A lot of previous versions are getting touched here. These do not belong in this commit. Also is there the intention to add a max line length parameter to .mdx files?

Reply:
I will correct this. I didn't intend to establish a maximum length in MDX files, although IntelliJ formats according to the same max line length as our Java code. I just missed the changes leaking into the release commit.

That being said, any new MD/MDX I create generally does adhere to the same max line length requirement as our Java code.

Reply:
Fixed